draft-ietf-tls-tls13.md

```diff
@@ -1415,7 +1413,7 @@ one extension (at least "key_share" or "pre_shared_key").
 max_supported_version
 : The latest (highest valued) version of the TLS protocol offered by the
   client. This SHOULD be the same as the latest version supported. For this
-  version of the specification, the version will be { 3, 4 }. (See
+  version of the specification, the version ID will be { 3, 4 }. (See
```

Comment:
It's not inane, it's pedantic. This is a mess, so I'm trying to be explicit as possible.

As to struct vs enum, yeah, I originally wanted to just use a uint16, but I kept things with this notation here to avoid changing too much in one PR. I'll add a second commit here that changes the notation.